[2023-09-17T19:16:45.965+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: your_python_dag.load_data_task manual__2023-09-17T19:16:42.686346+00:00 [queued]>
[2023-09-17T19:16:45.973+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: your_python_dag.load_data_task manual__2023-09-17T19:16:42.686346+00:00 [queued]>
[2023-09-17T19:16:45.974+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-09-17T19:16:45.974+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-09-17T19:16:45.974+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-09-17T19:16:45.985+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): load_data_task> on 2023-09-17 19:16:42.686346+00:00
[2023-09-17T19:16:46.007+0000] {standard_task_runner.py:55} INFO - Started process 3487 to run task
[2023-09-17T19:16:46.015+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'your_python_dag', 'load_data_task', 'manual__2023-09-17T19:16:42.686346+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmp647btxxz']
[2023-09-17T19:16:46.021+0000] {standard_task_runner.py:83} INFO - Job 50: Subtask load_data_task
[2023-09-17T19:16:46.138+0000] {task_command.py:388} INFO - Running <TaskInstance: your_python_dag.load_data_task manual__2023-09-17T19:16:42.686346+00:00 [running]> on host 80c059a34b8e
[2023-09-17T19:16:46.225+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=your_name
AIRFLOW_CTX_DAG_ID=your_python_dag
AIRFLOW_CTX_TASK_ID=load_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-09-17T19:16:42.686346+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-09-17T19:16:42.686346+00:00
[2023-09-17T19:16:46.261+0000] {python.py:177} INFO - Done. Returned value was:      CUST_ID      BALANCE  ...  PRC_FULL_PAYMENT  TENURE
0     C10001    40.900749  ...          0.000000      12
1     C10002  3202.467416  ...          0.222222      12
2     C10003  2495.148862  ...          0.000000      12
3     C10004  1666.670542  ...          0.000000      12
4     C10005   817.714335  ...          0.000000      12
...      ...          ...  ...               ...     ...
8945  C19186    28.493517  ...          0.500000       6
8946  C19187    19.183215  ...          0.000000       6
8947  C19188    23.398673  ...          0.250000       6
8948  C19189    13.457564  ...          0.250000       6
8949  C19190   372.708075  ...          0.000000       6

[8950 rows x 18 columns]
[2023-09-17T19:16:46.305+0000] {xcom.py:635} ERROR - Object of type DataFrame is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-09-17T19:16:46.307+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2297, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 240, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-09-17T19:16:46.323+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=your_python_dag, task_id=load_data_task, execution_date=20230917T191642, start_date=20230917T191645, end_date=20230917T191646
[2023-09-17T19:16:46.334+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 50 for task load_data_task (Object of type DataFrame is not JSON serializable; 3487)
[2023-09-17T19:16:46.386+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-09-17T19:16:46.414+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
